docker/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM ghcr.io/puppeteer/puppeteer:17.0.0
```

Comment:
Currently, we are using Puppeteer version 15.4.0. I don't mind upgrading to the latest version but we need to update the version on the package.json as well.

Please note that some tests might break because the rendering slightly change from version to version (mostly related to fonts).

Reply:
So as a first step I would use 15.4.0 and then upgrade to 17.0.0 as a follow-up.

Reply:
I have no preference about the versions, but understand the importance of the road map. 

Sadly the [puppeteer versions](https://github.com/orgs/puppeteer/packages/container/package/puppeteer) have only 16.1.0 as earliest version. 

So using their base image is not possible. I could adapt the Dockerfile to install puppeteer and Google Chrome. This needs proper testing from my side. I will look into it.

Reply:
I have rewritten the Dockerfile from Scratch, you we use the dependency version.